website/src/pages/documents/document.page.tsx

```diff
@@ -82,6 +83,11 @@ const TabSet = ({ doc }: { doc: Document }) => {
           </Tab>
         </TabList>
       </div>
+      <Button id="scroll-top" className={css.scrollTop} onClick={() => window.scrollTo({top: 0, behavior: 'smooth'})}>
+        <div><a><RiArrowUpCircleFill size={45}/></a></div>
+        <div>Top</div>
```

Comment:
What's going on here? A few suggestions: you don't need `<div>` tags everywhere you put content. I don't think you need either here. You also don't need the `<a>` tag which creates a link because this is already inside of a button. Finally, for the text, I'd suggest something that describes the action being performed, like "Scroll to Top". Especially if we only show the text part on desktop, the space difference should be negligible.